### PR TITLE
The updation of shields is needed in README.md

### LINKED ISSUE #222: [Docs]The updation of shields is needed in README.md
**Current State:**
Small updation of shields in README.md

image

**Desired State:**
This is how it should look like

image

updation should be done in this README.md by keeping the shield style uniform to following meshery-istio's README.md :
https://github.com/meshery/meshery-istio/blob/721b78783182e65145cefde64d905dab3ca20ba1/README.md?plain=1#L13

---
**Contributor Resources**
- [Meshery documentation site](https://docs.meshery.io/)
- [Meshery documentation source](https://github.com/meshery/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by msaini0r**
**Description**
 this pr solves the updation of shields is needed in README.md .

This PR fixes #222 

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by msaini0r** [comment]
@theBeginner86 can you review this pull request also.

File: .github/workflows/newcomers-alert.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Newcomers Alert
```
Comment (theBeginner86) [inline .github/workflows/newcomers-alert.yml:1]: I believe this file was not required for this PR

File: .github/workflows/slack.yml [hunk 2]
```diff
@@ -5,13 +5,12 @@ jobs:
...
-      run: |
-        echo ::set-env name=STARS::$(curl --silent 'https://api.github.com/repos/meshery/meshery-kuma' -H 'Accept: application/vnd.github.preview' | jq '.watchers_count')
-    - name: Notify slack
-      env:
-        SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
-      uses: pullreminders/slack-action@master
-      with:
-        args: '{\"channel\":\"CSK7N9TGX\",\"text\":\"${{ github.actor }} just starred Meshery Adapter for Kuma! (https://github.com/meshery/meshery-kuma/stargazers) Total ⭐️: ${{env.STARS}}\"}'
-        
+      - name: Get current star count
```
Comment (theBeginner86) [inline .github/workflows/slack.yml:8]: I believe this file was not required for this PR

**Review by theBeginner86 — CHANGES_REQUESTED** [review]
@msaini0r 

Changes to README.md looks good👍

I feel you have added extra files in this PR. Could you please remove them?

**Pull request closed without merging**